source/extensions/filters/http/ext_proc/headers.cc

```diff
@@ -0,0 +1,45 @@
+#include "extensions/filters/http/ext_proc/headers.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace ExternalProcessing {
+
+using Http::LowerCaseString;
+
+void buildHttpHeaders(const Http::HeaderMap& headers_in,
```

Comment:
Curious why there aren't unit tests; I had assumed from the code structure that you had split this out for unit testability.

Reply:
I had good test coverage and neglected it. I discovered more stuff that this function should do so the next commit makes it more complete and adds unit tests.